sdks/python/setup.py

```diff
@@ -19,24 +19,23 @@ def read(fname):
 
 setup(
     name="keelson",
-    version="0.3.6",
+    version="0.3.7",
     license="Apache License 2.0",
     description="A python Software Development Kit for keelson",
     long_description=read("README.md"),
     long_description_content_type="text/markdown",
-    url="https://github.com/MO-RISE/keelson/sdks/python",
+    url="https://github.com/rise-maritime/keelson/sdks/python",
     author="Fredrik Olsson",
     author_email="fredrik.x.olsson@ri.se",
     maintainer="Fredrik Olsson",
     maintainer_email="fredrik.x.olsson@ri.se",
     packages=find_namespace_packages(exclude=["tests", "dist", "build"]),
     python_requires=">=3.7",
     install_requires=[
-        "eclipse-zenoh>=0.11.0rc3",
-        "protobuf",
+        "eclipse-zenoh==0.11.0rc3",
+        "protobuf==5.28.0",
```

Comment:
Here is the source of the problem you have with the "UserWarning: protobuf gencode version 5.27.0 is older ...". The python package "protobuf" does not have the capability to compile .proto files. The .proto files are compiled with "protoc" which is installed as a python-wheel through the package "protoc-wheel-0" in "requirements_dev.txt". That python-wheel file contains protoc version 5.27, which is an older version than the protobuf version you install. So there is a mismatch between the version of the compiled files and the version of the software used to read them. Unfortunately, there is no similar python package for protoc version 5.28, so to fix the error we would have to stop using "protoc-wheel-0". This could be solved by developing inside of a container and installing protoc v5.28 in that container. I can help you out with that after this pull request is done.